Catappult/appcoins-wallet-android
# Eskl 163/allowed username chars

**What does this PR do?**

   Prevents url parameter injection.
   Validates usernames.
   Adds new Result codes.

**Database changed?**

   No

**Where should the reviewer start?**

- [ ] SkillsFragment.kt
- [ ] SkillsViewModel.kt
- [ ] UriValidationResult.kt

**How should this be manually tested?**

  Run on debug mode with a breakpoint wherever necessary, for example, in the finishWithError method, to check if wallet sends appropriate Result code.

**What are the relevant tickets?**

  Tickets related to this pull-request: [APPC-163](https://aptoide.atlassian.net/browse/APPC-163)

**Code Review Checklist**
